components/sup-protocol/protocols/ctl.proto

```diff
@@ -104,6 +104,11 @@ message SvcLoad {
 // Request to unload a loaded service.
 message SvcUnload {
   optional sup.types.PackageIdent ident = 1;
+  // Name of the signal to send the service to shut it down (e.g.,
+  // "TERM" and not "SIGTERM"). Only applies to Unix platforms.
+  optional string signal = 2;
+  // Timeout in seconds before killing the service
+  optional uint32 timeout = 3;
```

Comment:
I'd prefer
```suggestion
  optional uint32 timeout_in_secs = 3;
```
and ditch the comment. Similarly elsewhere.

Reply:
Yup